rero/rero-ils
# translation: fix key source issues

When translating on transifex, translators identify some source strings
that have to be improved or amended. This commit fixes a good part of
the current issues. The description fields of the schema of patron
transaction and patron transaction event have been removed in order to
avoid the need to translate them on transifex.

* Updates sources files.
* Updates source language files.
* Update translation files.
* Extracts messages.
* Updates catalogs.

Co-Authored-by: Igor Milhit <igor.milhit@rero.ch>

## Why are you opening this PR?

To fix source string issues identified on transifex.

## How to test?

- Check changes.
- Run tests.

## Code review check list
